src/goSuggest.ts

```diff
@@ -50,8 +50,22 @@ export class GoCompletionItemProvider implements vscode.CompletionItemProvider {
 				let lineTillCurrentPosition = lineText.substr(0, position.character);
 				let autocompleteUnimportedPackages = config['autocompleteUnimportedPackages'] === true && !lineText.match(/^(\s)*(import|package)(\s)+/);
 
-				if (lineText.match(/^\s*\/\//)) {
-					return resolve([]);
+				if (lineText.match(/\//)) {  // complete exported item`s comment
+					if (!lineText.match(/^\/\//)) {
+						return resolve([]);
+					}
+					let nextLineText = document.lineAt(position.line + 1).text;
+					if (nextLineText) {
+						let tokens = nextLineText.split(' ');
```

Comment:
When there are extra spaces between `func` and the name of the function, this will result in empty text being suggested

Reply:
Is having other util for picking up function`s name?

Reply:
I would suggest to use regular expressions

`/^\s*type\s+([A-Z]\S*)/` for types
`/^\s*func\s+\(\s*\w+\s+\*?\w+\s*\)\s+([A-Z]\S*)/` for methods with receivers
`/^\s*func\s+([A-Z]\S*)/` for functions

When you use `let matches = nextLineText.match()` on the above, `matches[1]` should give you the symbol name